.mailmap

```diff
@@ -548,3 +550,13 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
 Vaibhav Bhat <vaibhav.bhat2097@gmail.com> VBhat97 <vaibhav.bhat2097@gmail.com>
 Bhaskar Joshi <bhaskar.joshi@research.iiit.ac.in> BhaskarJoshi-01 <bhaskar.joshi@research.iiit.ac.in>
 Miguel Torres Costa <miguelptcosta1995@gmail.com>
+Elisha Hollander <just4now666666@gmail.com> donno2048 <just4now666666@gmail.com>
+Mamidi Ratna Praneeth <praneethratna@gmail.com> praneeth <praneethratna@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com> Zach <20629897+craymichael@users.noreply.github.com>
+Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
+Harshit Yadav <harshityadav2k@gmail.com> Harshit Yadav <45384915+hyadav2k@users.noreply.github.com>
+Megan Ly <megan@artcompiler.com> Megan Ly <meganly@MacBook-Pro.local>
+Robert Pollak <robert.pollak@jku.at> Robert Pollak <robert.pollak@posteo.net>
+gum3ng <gum3ng@protonmail.com> gum3ng <78410304+gum3ng@users.noreply.github.com>
+mostlyaman <amansharma110603@gmail.com> mostlyaman <85072876+mostlyaman@users.noreply.github.com>
```

Comment:
I have emailed asking what name to use

Reply:
Hi, I've replied to your mail. Please use Aman Sharma as my name. Thanks.

Reply:
This is
```
Aman Sharma <amansharma110603@gmail.com>
```